fix/basics-cho-api (#2318)
mercadopago/devsite-docs: guides/checkout-api-v2/create-customer-and-card.en.md

To create a customer and associate it with your card, you must send the email field, the type of payment method, the ID of the issuing bank and the generated token. Each customer will be saved with the value `customer` and each card with the value `card`.

> In addition, by using our SDKs, you can store your card data whenever a payment is successfully completed. This allows the correct data to be stored securely in our servers for future purchases, and streamlines the payment process for the buyer. For more information, check our [API Reference](/developers/en/reference/cards/_customers_customer_id_cards/post).

Author: ```suggestion
In addition, by using our SDKs, and if a payment is successfully completed, the correct information can be stored securely in our servers for future purchases, what streamlines the payment process for the buyer. For more information, check our [API Reference](/developers/en/reference/cards/_customers_customer_id_cards/post).
```
